ui/components/MesheryPatterns/MesheryPatternGridView.js

```diff
@@ -234,9 +217,21 @@ function MesheryPatternGrid({ patterns=[], handleVerify, handlePublish, handleUn
         validationBody={modalOpen.validationBody}
       />
       {canPublishPattern &&
-      <Modal open={publishModal.open} schema={publish_schema} uiSchema={publish_ui_schema} onChange={onChange} handleClose={handlePublishModalClose} formData={_.isEmpty(payload.catalog_data)? publishModal?.pattern?.catalog_data : payload.catalog_data} aria-label="catalog publish" title={publishModal.pattern?.name} payload={payload} handleSubmit={handlePublish} showInfoIcon={{ text : "Upon submitting your catalog item, an approval flow will be initiated.", link : "https://docs.meshery.io/concepts/catalog" }} />
+       <Modal
+         open={publishModal.open}
+         schema={publish_schema}
+         uiSchema={publish_ui_schema}
+         onChange={onChange}
+         handleClose={handlePublishModalClose}
+         aria-label="catalog publish"
+         title={publishModal.pattern?.name}
+         handleSubmit={handlePublish}
+         payload={payload}
```

Comment:
```suggestion
         handleClose={handlePublishModalClose}
         title={publishModal.pattern?.name}
         handleSubmit={handlePublish}
```

Reply:
after merging this change I think we're good to go or anything left @Abhishek-kumar09

Reply:
MeshMap.